### DISCUSSION TITLE
help with data input

### DISCUSSION THREAD

**Discussion by likeaRAINBOW** [Lurkars/gloomhavensecretariat#423] [Q&A]
hey,

i hope it's the right place for this.
So, i saw that data input may an issue for you, so my question is, do you need some help with that?

**Comment by Lurkars** [comment]
Yes of course. Problem for now: most missing data now is events, where I still need to develop best format to handle all use cases. So in the end doesn't make sense. If you really want to help now, then you can grab the Crimson Scales books and work through all scenario to check for missing/wrong data/special rules.
